app/models/spectra_set.rb

```diff
@@ -8,7 +8,7 @@ class SpectraSet < ActiveRecord::Base
   has_and_belongs_to_many :spectrums
   belongs_to :user
 
-  validates :title, :format => { with: /\A[\w\ -\'\"]+\z/, message: "can contain only letters, numbers, and spaces." }
```

Comment:
Was this change needed?
If so then please change error message !

Reply:
Oh sure, I'll change the title, the regex change was to accommodate more symbols in the title